package.json

```diff
@@ -12,6 +12,8 @@
     "postinstall": "npm run postinstall:nx && npm run postinstall:ngcc",
     "postinstall:ngcc": "npm run ngcc",
     "postinstall:nx": "node ./decorate-angular-cli.js",
+    "check:dotnet": "dotnet --version || echo ❌ dotnet CLI must be installed",
+    "check:java": "java -version || echo ❌ Java must be installed",
```

Comment:
question: What is Java used for?

Reply:
Java is used for the OpenAPI-to-Angular HTTP client generation.